### DISCUSSION TITLE
when will bf16 precision be supported on official release?

### DISCUSSION THREAD

**Discussion by grudloff** [Lightning-AI/pytorch-lightning#9527] [General]
Hello! The docs for [mixed-precision](https://pytorch-lightning.readthedocs.io/en/latest/advanced/mixed_precision.html?highlight=BFloat16%20Mixed%20Precision#bfloat16-mixed-precision) seemed to indicate that `precision='bf16'` was a feature already supported, that just required to install `pytorch-nightly`. But it seems that PR #9049, which implements this, is available only by building from source. When is it planned to be added?

**Comment by tchaton** [comment]
Dear @grudloff,

PyTorch would make 1.1.0 RC on Sept 21 and the release on Oct 21.
PyTorch Lightning is going to align its release on PyTorch and we will make our first RC on Sept 28 and release on Oct 26.

Those dates might slightly change :) Hope it helps.

Best,
T.C
